tests/acceptance/30_generic_methods/file_ensure_lines_present_test_already_present.cf

```diff
@@ -48,16 +48,14 @@ bundle agent test
 bundle agent check
 {
   vars:
-    "line_exists_test" string => "/bin/grep \"^${init.line_to_add}$\" \"${init.file}\" | wc -l";
-    "nb_lines" 
-      string     => execresult("${line_exists_test}", "useshell"),
-      ifvarclass => canonify("file_ensure_lines_present_${init.file}_reached");
+    "line_exists_test" string => "/usr/bin/test `/bin/grep \"^${init.line_to_add}$\" \"${init.file}\" | wc -l` = '1'";
 
   classes:
     # By default, file_ensure_lines_present should create the file if it doesn't exist
     "file_exists" expression => fileexists("${init.file}");
     
-    "line_exists" expression => strcmp("${nb_lines}", "1");
```

Comment:
The strings were not equal on AIX because ```wc -l``` returns a lot of leading spaces. Doing the comparison in shell means these spaces are ignored.